class.jetpack.php

```diff
@@ -5310,6 +5310,27 @@ public static function is_staging_site() {
 	}
 
 	/**
+	 * Returns the value of the jetpack_sync_idc_optin filter, or constant.
+	 * If set to true, the site will be put into staging mode.
+	 *
+	 * @since 4.4.0
+	 * @return bool
+	 */
+	public static function sync_idc_optin() {
+		/**
+		 * Allows sites to optin to IDC mitigation which blocks the site from syncing to WordPress.com when the home
+		 * URL or site URL do not match what WordPress.com expects.
+		 * The default value is either false, or the value of JETPACK_SYNC_IDC_OPTIN constant if set
```

Comment:
Minor, but we should probably add a period after `set`. And maybe either move the second sentence up a line or add an empty line?